### PR TITLE
gateway follow up

### PR DESCRIPTION
**Pull request by jburel**
Indicate that the gateway is still under development.

cc @joshmoore @dominikl

### CODE REVIEW

File: omero/developers/whatsnew.txt [hunk 1]
```diff
@@ -44,6 +44,16 @@ Java Gateway
...
+See :doc:`/developers/Java` for more information or follow the example in
+`minimal-omero-client <https://github.com/ome/minimal-omero-client>`_.
+
+OMERO model
+^^^^^^^^^^^
+
+The :literal:`Rect` class has been renamed to :literal:`Rectangle` to match
+the OME-XML schema. This requires the corresponding change to be made in
+client software that uses the OMERO.blitz server API to work with ROIs.
+>>>>>>> 987f3d4... add what's new note about Rect → Rectangle
```
Comment (mtbc) [inline omero/developers/whatsnew.txt]: whoops

File: omero/developers/whatsnew.txt [hunk 2]
```diff
@@ -44,6 +44,15 @@ Java Gateway
 ^^^^^^^^^^^^
 
 The Java gateway has undergone a major overhaul. 
-This should make the development of Java applications much easier. See
-:doc:`/developers/Java` for more information or follow the example in
-`minimal-omero-client <https://github.com/ome/minimal-omero-client>`_.
\ No newline at end of file
+This should make the development of Java applications much easier.
+We do not intend to modify the methods already available but this work should
+still be considered as under development. Feedback as always welcome.
```
Comment (hflynn) [inline omero/developers/whatsnew.txt]: as -> is ?

**Comment by hflynn** [comment]
minor quibble but works for me otherwise

**Comment by jburel** [comment]
@hflynn: fixed

**Comment by hflynn** [comment]
Thanks

**Pull request merged**